### PR TITLE
add code of conduct 2.0 [https://github.com/MarquezProject/marquez/is…

### PR DESCRIPTION
**Pull request by sreev**
…sues/850]

  https://github.com/MarquezProject/marquez/issues/850

Signed-off-by: Sree Vaddi <441385+sreev@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ Marquez is an open source **metadata service** for the **collection**, **aggrega
 
 Marquez is an [LF AI & Data Foundation](https://lfaidata.foundation/projects/marquez) incubation project under active development and we'd love your help!
 
+## Code of Conduct
```
Comment (wslulciuc) [inline README.md:22]: With the contributor covenant badge added to our _Badges_ section, this section can be removed.

**Review by wslulciuc — CHANGES_REQUESTED** [review]
This looks good, I'd remove the _Code of Conduct_ (in the README.md) as we already link out to the contributor covenant via the badge. The section would make more sense to be added to our [CONTRIBUTING.md](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md)

**Comment by sreev** [comment]
code of conduct should and must be the first read.

**Comment by wslulciuc** [comment]
> We recommend that you add the Markdown or text version of the Contributor Covenant to your source code repository at the root level.

They just recommend we add the text to the root of our repo, which we are with [CODE_OF_CONDUCT.md](https://github.com/MarquezProject/marquez/blob/main/CODE_OF_CONDUCT.md). Am I missing something?

File: README.md [hunk 2]
```diff
@@ -19,6 +19,15 @@ Marquez is an open source **metadata service** for the **collection**, **aggrega
...
+We are proud to be an adopter of [Contributor Covenant](https://www.contributor-covenant.org/).
+_Pledge your respect and appreciation for contributors of all kinds to your open source project._
+
+There are 184 two-letter language codes ([ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1)) registered as of December 2018.
+We’re always looking for new localizations of Contributor Covenant and are thankful to the volunteers who spend their time on translations.
+
+Instructions for translations can be found [here](https://github.com/ContributorCovenant/contributor_covenant#translating).
```
Comment (julienledem) [inline README.md:29]: I don't think this section is relevant to the Marquez README.
Having the badge and CODE_OF_CONDUCT.md at the root of the repo is sufficient.

**Review by julienledem — CHANGES_REQUESTED** [review]
Thanks for updating the code of conduct to the latest version.
Please adress my comment bellow so that we can merge.

**Comment by wslulciuc** [comment]
Closing as this code of conduct has been updated.

**Pull request closed without merging**